db/migrate/20221227083132_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
```

Comment:
```suggestion
      t.string :email, null: false
```

FYI
実際にはこういった形でバリデーションしておくことがほとんどです
今回はこのままで問題なしです 👍